addons/mrp/tests/test_warehouse_multistep_manufacturing.py

```diff
@@ -609,3 +609,43 @@ def test_child_parent_relationship_on_backorder_creation(self):
         self.assertEqual(mo.mrp_production_child_count, 0, "Children MOs counted as existing where there should be none")
         self.assertEqual(mo.mrp_production_source_count, 0, "Source MOs counted as existing where there should be none")
         self.assertEqual(mo.mrp_production_backorder_count, 2)
+
+    def test_split_merge_3_steps(self):
```

Comment:
You still need to rename this test. You are not splitting any MOs in this test.

Reply:
@ticodoo  oh, forgot that done that change please check it.